.gitignore

```diff
@@ -11,3 +11,5 @@ playground
 tmp
 dist
 .turbo
+**/.serena
```

Comment:
I added these because my bot would have dumped a load of memory files you may not want, and this is a common/popular agent framework that people should be able to/allowed to use on this project with out polluting the git.